### PR TITLE
Create sim_coherenceOrder.m

### PR DESCRIPTION
**Pull request by goedan-cic**
generates the coherence order matrix for a spin system for use in sim_Hamiltonian

### CODE REVIEW

File: simulationTools/sim_coherenceOrder.m [hunk 1]
```diff
@@ -0,0 +1,31 @@
+%sim_coherenceOrder.m
+%Dana Goerzen and Jamie Near, 2021.
+%
+% USAGE:
+% out = sim_coherenceOrder(spinSys);
+%
+% DESCRIPTION:
+% Creates a n x n matrix corresponding to the coherence order of each
+% proton in spin system. For use in nulling signal incoherences during
```
Comment (jamienear) [inline simulationTools/sim_coherenceOrder.m:9]: I would rephrase this first sentence to say that it "Creates an n x n matrix that represents the coherence order of each element of the density matrix for the given spin system."

**Review by jamienear — CHANGES_REQUESTED** [review]
Hi Dana, this looks great.  Only one suggested change in the comment description of the function.

File: simulationTools/sim_coherenceOrder.m (hunk 1 above)
Comment (goedan-cic) [reply simulationTools/sim_coherenceOrder.m]: Done

**Review by jamienear — APPROVED** [review]
